docs/docs/keymaps/behaviors/mouse-emulation.md

```diff
@@ -69,3 +75,127 @@ This example will send press of the fourth mouse button when the binding is trig
 ```
 &mkp MB4
 ```
+
+### Input Processors
+
+If you want to apply any [input processors](../input-processors/index.md#input-processors-overview) to `&mkp` you can do so by referencing `&mkp_input_listener`, e.g.:
+
+```dts
+&mkp_input_listener {
+    input-processors = <&zip_temp_layer 2 2000>;
+}
+```
+
+## Mouse Move
+
+This behavior sends mouse X/Y movement events to the connected host.
+
+### Behavior Binding
+
+- Reference: `&mmv`
+- Parameter: A `uint32` with 16-bits each used for vertical and horizontal max velocity.
+
+The following predefined values can be passed for the parameter:
+
+| Define       | Action     |
+| :----------- | :--------- |
+| `MOVE_UP`    | Move up    |
+| `MOVE_DOWN`  | Move down  |
+| `MOVE_LEFT`  | Move left  |
+| `MOVE_RIGHT` | Move right |
```

Comment:
Another option to surface defaults would be to add a `Default value` column here and for the `SCRL_` defines.

Reply:
I feel like this would be a bit annoying with duplicates, unless you wanted to fully replicate the value in each column like `MOVE_Y(-ZMK_MOUSE_DEFAULT_MOVE_VAL)` for `MOVE_UP`, etc. but that seems more confusing than helpful, IMHO.

Reply:
I was thinking `MOVE_Y(-600)` etc., but the existence of `ZMK_MOUSE_DEFAULT_MOVE_VAL` which can also be modified by the user is complicating it a bit. I'll add another suggestion for the paragraph below, instead.